<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>hotelRegister</class>
 <widget class="QDialog" name="hotelRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1227</width>
    <height>664</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>酒店及管理员注册</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/myimages/icon/注册页_入驻成功 #d8432a &amp; 66px.png</normaloff>:/myimages/icon/注册页_入驻成功 #d8432a &amp; 66px.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>管理员信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>20</y>
     <width>151</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>酒店信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>240</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 20pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>客房信息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>302</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>张海山锐谐体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>302</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>张海山锐谐体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>定价</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>302</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>张海山锐谐体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>折扣</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>302</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>张海山锐谐体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>房型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>302</y>
     <width>40</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>张海山锐谐体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>描述</string>
   </property>
  </widget>
  <widget class="QLabel" name="hotelName">
   <property name="geometry">
    <rect>
     <x>621</x>
     <y>111</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>酒店名： </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hotelNameInput">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>111</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入酒店名</string>
   </property>
  </widget>
  <widget class="QLabel" name="hotelAddress">
   <property name="geometry">
    <rect>
     <x>622</x>
     <y>151</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>酒店地址： </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cityInput">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>151</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入城市名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="districtInput">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>151</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入地区名</string>
   </property>
  </widget>
  <widget class="QLabel" name="hotelPhonenum">
   <property name="geometry">
    <rect>
     <x>621</x>
     <y>191</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>酒店电话： </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>711</x>
     <y>191</y>
     <width>16</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="hotelPhonenumInput">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>191</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入酒店电话</string>
   </property>
  </widget>
  <widget class="QLabel" name="phonenumLabel">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>301</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>手机号： </string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>202</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>用户名： </string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>251</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>密码： </string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>449</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>身份证号： </string>
   </property>
  </widget>
  <widget class="QLabel" name="sexLabel">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>399</y>
     <width>53</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>性别： </string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>62</x>
     <y>350</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>真实姓名： </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="phonenumInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>300</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入11位手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入18位身份证号</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sexComboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>350</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入真实姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="singleLabel">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>352</y>
     <width>68</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>单人间： </string>
   </property>
  </widget>
  <widget class="QLabel" name="doubleLabel">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>394</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>双人间： </string>
   </property>
  </widget>
  <widget class="QLabel" name="tripleLabel">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>437</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>三人间： </string>
   </property>
  </widget>
  <widget class="QLabel" name="kingSizeLabel">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>481</y>
     <width>68</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>大床间： </string>
   </property>
  </widget>
  <widget class="QLabel" name="suiteLabel">
   <property name="geometry">
    <rect>
     <x>391</x>
     <y>524</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>套间：  </string>
   </property>
  </widget>
  <widget class="QSpinBox" name="singleSpinBox">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>352</y>
     <width>52</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>394</y>
     <width>52</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="tripleSpinBox">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>436</y>
     <width>52</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="kingSizeSpinBox">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>478</y>
     <width>52</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="suiteSpinBox">
   <property name="geometry">
    <rect>
     <x>507</x>
     <y>520</y>
     <width>52</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="singleFixedPriceInput">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>352</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleFixedPriceInput">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>394</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tripleFixedPriceInput">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>436</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kingSizeFixedPriceInput">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>478</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="suiteFixedPriceInput">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>520</y>
     <width>108</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>100000.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="singleDiscountInput">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>355</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleDiscountInput">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>396</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="tripleDiscountInput">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>437</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kingSizeDiscountInput">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>478</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="suiteDiscountInput">
   <property name="geometry">
    <rect>
     <x>791</x>
     <y>519</y>
     <width>68</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLineEdit" name="singleDescribeInput">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>356</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入单人间描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doubleDescribeInput">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>397</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入双人间描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tripleDescribeInput">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>438</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入三人间描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kingSizeDescribeInput">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>479</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入大床间描述</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="suiteDescribeInput">
   <property name="geometry">
    <rect>
     <x>921</x>
     <y>520</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入套间描述</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>590</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);
font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>确认注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>590</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 12pt &quot;3ds Light&quot;;
border-radius: 3px;
border: 1px solid rgb(0, 0, 0);
font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>usernameInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>phonenumInput</tabstop>
  <tabstop>nameInput</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>idInput</tabstop>
  <tabstop>hotelNameInput</tabstop>
  <tabstop>cityInput</tabstop>
  <tabstop>districtInput</tabstop>
  <tabstop>hotelPhonenumInput</tabstop>
  <tabstop>singleSpinBox</tabstop>
  <tabstop>singleFixedPriceInput</tabstop>
  <tabstop>singleDiscountInput</tabstop>
  <tabstop>singleDescribeInput</tabstop>
  <tabstop>doubleSpinBox</tabstop>
  <tabstop>doubleFixedPriceInput</tabstop>
  <tabstop>doubleDiscountInput</tabstop>
  <tabstop>doubleDescribeInput</tabstop>
  <tabstop>tripleSpinBox</tabstop>
  <tabstop>tripleFixedPriceInput</tabstop>
  <tabstop>tripleDiscountInput</tabstop>
  <tabstop>tripleDescribeInput</tabstop>
  <tabstop>kingSizeSpinBox</tabstop>
  <tabstop>kingSizeFixedPriceInput</tabstop>
  <tabstop>kingSizeDiscountInput</tabstop>
  <tabstop>kingSizeDescribeInput</tabstop>
  <tabstop>suiteSpinBox</tabstop>
  <tabstop>suiteFixedPriceInput</tabstop>
  <tabstop>suiteDiscountInput</tabstop>
  <tabstop>suiteDescribeInput</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>returnButton</sender>
   <signal>clicked()</signal>
   <receiver>hotelRegister</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
